pkg/k8splan/watcher.go

```diff
@@ -51,15 +59,23 @@ func (w *watcher) start(ctx context.Context) {
 		panic(err)
 	}
 
+	if err := validateKC(ctx, kc); err != nil {
+		if strings.Contains(err.Error(), "x509: certificate signed by unknown authority") && len(kc.CAData) != 0 {
+			logrus.Debugf("Initial connection to Kubernetes cluster failed with error %v, removing CA data and trying again", err)
```

Comment:
maybe this important enough to be an info log?

Reply:
moved to info log